src/Common/PasswordManipulation.php

```diff
@@ -59,16 +59,11 @@ public function getLength()
      *
      * @since 2.9.7
      *
-     * @return mix-data
+     * @return mixed
```

Comment:
`generatePassword` will not only return `string` type?

Reply:
it technically string, but with combinatin of specialChars, numbers etc

Reply:
is am right?